build/Publish.targets

```diff
@@ -276,6 +280,10 @@
         ManifestArtifactData="NonShipping=true"
         Condition="'%(PackageToPublish.Category)' != 'ship'" />
 
+      <PackageToPublishToTransport
```

Comment:
`PublishToTransportFeed` is the target that runs when we're trying to publish to the orchestrated feed, so as long as I also update https://devdiv.visualstudio.com/DevDiv/_git/DotNet-Orchestration-Utilities?path=%2Fsrc%2FFinal.VSFeedPublish.targets&version=GBmaster, the orchestration should take care of copying the redist packages from the orchestrated feed to the VS feed

Reply:
Another option would be to include these packages in `@(FilesToPublish)` i.e. changing the input `GetFilesToPublish` target. That has the advantage of placing the files in the \\aspnetci drop too.